### ISSUE TITLE
Group : Incorrect use of fields on the UI to underlying storage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#747]
```
Groups -- 

short_name is presented as "Acronym"

        Field(name="short_name", # [user-req]
            modes="view edit add listing",
            localizable=[
                show("view edit"),
                hide("listing"),
            ],
            property=schema.TextLine(title=_("Acronym")),
            #listing_column=name_column("short_name", _("Name"))
        ),


full_name is presented as "Name" 

Field(name="full_name", # [user-req]
            modes="view edit add listing",
            localizable=[
                show("view edit"),
                hide("listing"),
            ],
            property=schema.TextLine(title=_("Name")),
            #listing_column=name_column("full_name", _("Full Name"))
        ),



Parliaments (which is derived from Group)

short_name is used as "Parliament Identifier"

Field(name="short_name", # [user-req]
            modes="view edit add listing",
            localizable=[
                show("view edit listing"),
            ],
            property=schema.TextLine(title=_("Parliament Identifier"),
                description=_("Unique identifier of each Parliament "
                    "(e.g. IX Parliament)"),
            ),
        ),


full_name is presented as "Name" 

fields = [
        Field(name="full_name", # [user-req]
            description=_("Parliament name"),
            modes="view edit add listing",
            localizable=[
                show("view edit listing"),
            ],
            property=schema.TextLine(title=_("Name")),
            listing_column=name_column("full_name", "Name"),
        ),



Government (which is derived from Group)

short_name is used as "Name" 

  Field(name="short_name", # [user]
            modes="view edit add listing",
            localizable=[
                show("view edit add listing"),
            ],
            property=schema.TextLine(title=_("Name"),
                description=_("Name"),
                required=False
            ),

full_name is used as "Number"

Field(name="full_name", label=_("Number"), # [user]
            modes="view edit add listing",
            localizable=[
                show("view edit add listing"),
            ],
        ),



In summary -- 

short_name is used in 3 different groups in different ways as :
"Acronym", (in Groups)
"Parliament Identifer" (in Parliament) and 
"Name" (in Government)

full_name is used in 2 different ways :
"Name" in Groups, Parliament (and other derived groups) 
"Number" in Government.

To FIX -- 

there seems to be a requirement for a "ACRONYM" / "IDENTIFER",

so we need to add a field called "acronym"  to groups which is used for 
"acronym" (Group) and "parliament identifer" (Parliament) and "Number" 
(Government) ...

short_name should be used as it is called "short name" or "short title"
full_name should be used as it is called "full name" or "long title"


```

Original issue reported on code.google.com by `ashok.ha...@gmail.com` on 31 Jan 2012 at 8:00
- Blocking: #754
